src/style.css

```diff
@@ -0,0 +1,83 @@
+* {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: Avenir-Book;
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 900;
+}
+
+body {
+  font-size: 16px;
+  font-family: Avenir-Book, Helvetica, sans-serif;
+  font-style: normal;
```

Comment:
Do you really need `font-style: normal` here? I think it's the default in any browser.

Reply:
Yes, you are right.